modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -79,7 +80,7 @@ export class Wallet implements IWallet {
   public readonly bitgo: BitGoBase;
   public readonly baseCoin: IBaseCoin;
   private _wallet: WalletData;
-  private readonly tssUtils: ITssUtils;
+  private readonly tssUtils: EcdsaUtils | EddsaUtils | undefined;
```

Comment:
For educational purposes, what's the logic in this change?

Reply:
ITssUtils is depracted. ITssUtils = EddsaUtils however tss now also supports EcdsaUtils